pkg/scheduler/framework/plugins/podtopologyspread/plugin.go

```diff
@@ -148,7 +148,7 @@ func (pl *PodTopologySpread) EventsToRegister() []framework.ClusterEventWithHint
 		{Event: framework.ClusterEvent{Resource: framework.Pod, ActionType: framework.All}, QueueingHintFn: pl.isSchedulableAfterPodChange},
 		// Node add|delete|update maybe lead an topology key changed,
 		// and make these pod in scheduling schedulable or unschedulable.
-		{Event: framework.ClusterEvent{Resource: framework.Node, ActionType: framework.Add | framework.Delete | framework.Update}, QueueingHintFn: pl.isSchedulableAfterNodeChange},
+		{Event: framework.ClusterEvent{Resource: framework.Node, ActionType: framework.Add | framework.Delete | framework.UpdateNodeLabel | framework.UpdateNodeTaint}, QueueingHintFn: pl.isSchedulableAfterNodeChange},
```

Comment:
It should be OK. Actually before https://github.com/kubernetes/kubernetes/pull/122195, we only has `UpdateNodeLabel` rather than `Update`. (was changed to `Update` to deal with [this issue](https://github.com/kubernetes/kubernetes/issues/122284))